### PR TITLE
Some minor fixes and updated Polish translation

### PR DESCRIPTION
**Pull request by piotrdrag**

### CODE REVIEW

File: src/guake/about.py [hunk 1]
```diff
@@ -43,5 +43,5 @@ def __init__(self):
         img = gtk.gdk.pixbuf_new_from_file(ipath)
         dialog.set_property('logo', img)
 
-        dialog.set_name('Guake Terminal')
+        dialog.set_name(_('Guake Terminal'))
```
Comment (Stibbons) [inline src/guake/about.py]: beware this won't work, you need to include '_' (https://travis-ci.org/Guake/guake/builds/62825994)

**Comment by Stibbons** [comment]
Thanks a lot for all your contributions :)

**Comment by Stibbons** [comment]
I have a small comment on one patch, overall it's good

File: src/guake/about.py (hunk 1 above)
Comment (piotrdrag) [inline src/guake/about.py]: Do you mean adding "from guake.common import _"?

Comment (Stibbons) [inline src/guake/about.py]: I guess so, this is an import that needs to be added in each file that use '_'

Comment (piotrdrag) [inline src/guake/about.py]: I'm afraid I don't know anything about programming. Let's see if this works.

Comment (Stibbons) [inline src/guake/about.py]: let's merge!

**Comment by piotrdrag** [comment]
Is there a reason why this wasn't merged?

**Comment by Stibbons** [comment]
Sorry, I closed without clicking on merge. Your change should have been integrated into c6e800a88a9086a671684f3ecd86873598797269. Can you check if all your translations are sync in pl.po?

**Pull request merged**